tests/Mongolid/ActiveRecordTest.php

```diff
@@ -372,48 +372,39 @@ public function testShouldGetDataMapper()
         $this->assertEquals($schema, $result->getSchema());
     }
 
-    /**
-     * @expectedException \Mongolid\Exception\NoCollectionNameException
-     */
     public function testShouldRaiseExceptionWhenHasNoCollectionAndTryToCallAllFunction()
     {
         $entity = new class() extends ActiveRecord {
         };
 
-        $this->assertNull($entity->getCollectionName());
-
+        $this->expectException(NoCollectionNameException::class);
         $entity->all();
     }
 
-    /**
-     * @expectedException \Mongolid\Exception\NoCollectionNameException
-     */
     public function testShouldRaiseExceptionWhenHasNoCollectionAndTryToCallFirstFunction()
     {
         $entity = new class() extends ActiveRecord {
         };
 
-        $this->assertNull($entity->getCollectionName());
-
+        $this->expectException(NoCollectionNameException::class);
         $entity->first();
     }
 
-    /**
-     * @expectedException \Mongolid\Exception\NoCollectionNameException
-     */
     public function testShouldRaiseExceptionWhenHasNoCollectionAndTryToCallWhereFunction()
     {
         $entity = new class() extends ActiveRecord {
         };
 
-        $this->assertNull($entity->getCollectionName());
-
+        $this->expectException(NoCollectionNameException::class);
         $entity->where();
     }
 
     public function testShouldGetCollectionName()
     {
         $entity = new class() extends ActiveRecord {
+            /**
```

Comment:
Can this rule be disabled for anonymous classes or tests directory ?

Reply:
@ravanscafi What do you think?

Reply:
@kukulich Maybe we can add an option for that for `SlevomatCodingStandard.TypeHints.TypeHintDeclaration`?

Reply:
I need to solve https://github.com/slevomat/coding-standard/issues/353 first.